src/d_main.c

```diff
@@ -244,7 +245,7 @@ void D_Display (void)
     }
 
   // save the current screen if about to wipe
-  if ((wipe = gamestate != wipegamestate))
+  if ((wipe = gamestate != wipegamestate) && (screen_melt || strictmode))
```

Comment:
Should we introduce a `STRICTMODE` macro for this, similar to the one used in the menu code?

Reply:
I'm thinking maybe I should remove the `STRICTMODE` macro? Or write more generic one somehow.

Reply:
I think the macro is fine. :shrug:

Reply:
Ok but how to name the new one? `NOTSTRICTMODE`? I'm confused here 😕

Reply:
Yes, sure, why not? 😉